cromshell

```diff
@@ -63,6 +63,8 @@ if [[ $( uname ) == "Darwin" ]] ; then
   PING_CMD='ping -c1 -W1000 -t10'
 fi
 
+alias curl='curl --connect-timeout ${CURL_CONNECT_TIMEOUT} --max-time ${CURL_MAX_TIMEOUT}'
```

Comment:
This is fine, but as an aside `alias` will only work in this script because we set them to be expanded earlier (`shopt -s expand_aliases`).